app/ui/src/app/common/ui-patternfly/syndesis-form-control.component.html

```diff
@@ -8,8 +8,8 @@
          [class.required-pf]="model.required"
          [ngClass]="[elementLabelClass, gridLabelClass]">
     {{ model.label }}
-    <div *ngIf="model.hint !== null"
-         [tooltip]="model.hint"
+    <div *ngIf="model.labelTooltip !== null"
```

Comment:
Bear in mind that this will also render the tooltip if `model.labelTooltip` is `undefined` or just an empty string. Perhaps you might want to make the implementation more resilient by just doing:

```html
<div *ngIf="model.labelTooltip"
  ...
```